cmd/manager-hub/main.go

```diff
@@ -22,6 +22,9 @@ import (
 
 	buildv1 "github.com/openshift/api/build/v1"
 	imagev1 "github.com/openshift/api/image/v1"
+	buildocpbuild "github.com/rh-ecosystem-edge/kernel-module-management/internal/build/ocpbuild"
+	signocpbuild "github.com/rh-ecosystem-edge/kernel-module-management/internal/sign/ocpbuild"
```

Comment:
How about we just call them `build` and `sign` in the alias? `buildocpbuild` and `signocpbuild` are a bit confusing.